packages/apostrophe/modules/@apostrophecms/rich-text-widget/ui/apos/components/AposRichTextWidgetEditor.vue

```diff
@@ -196,6 +203,8 @@ export default {
   emits: [ 'update', 'suppressWidgetControls' ],
   data() {
     return {
+      widgetId: createId(),
```

Comment:
I don't think you need auto generated widgetId here? You should auto generate always styleId and use it everywhere.